agent/consul/discoverychain/compile.go

```diff
@@ -12,8 +12,9 @@ import (
 
 type CompileRequest struct {
 	ServiceName       string
-	CurrentNamespace  string
-	CurrentDatacenter string
+	CurrentNamespace  string // TODO(rb): rename to EvaluateInNamespace
```

Comment:
Didn't you do this in one of your other PRs? Was it one of those that is being shelved for now?

Reply:
I can do it here before merging. I just noticed in the middle of this PR that it made more sense to use the more descriptive names here since there are now 2 datacenter values involved, and the API already uses the more descriptive names.

Reply:
done